src/builder/get_messages.rs

```diff
@@ -98,8 +98,21 @@ impl GetMessages {
     ///
```

Comment:
There should be a mention about this change in the documentation that by using this method, the message cache will be populated with messages generated by it.